content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
-as well as over the network (if enabled).
+## Get started
+To get a better idea of what LXD is and what it does, you can [try it online](/lxd/try-it/)!
 
-Clients, such as the command line tool provided with LXD itself then do everything through that REST API.
-It means that whether you're talking to your local host or a remote server, everything works the same way.
+Then if you want to run it locally, take a look at our [getting started guide](/lxd/getting-started-cli/) or watch the following clip:
 
-# Features
-Some of the biggest features of LXD are:
+[![Installation and getting started](https://asciinema.org/a/226224.svg "Installation and getting started")](https://asciinema.org/a/226224)
+
+You can find a series of howtos and tutorials on YouTube:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLddduKsl-KEhleT9VTR4hbtlNdtMr6cFd" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
- * Secure by design (unprivileged containers, resource restrictions and much more)
- * Scalable (from containers on your laptop to thousand of compute nodes)
- * Intuitive (simple, clear API and crisp command line experience)
- * Image based (with a wide variety of Linux distributions published daily)
- * Support for Cross-host container and image transfer (including live migration with CRIU)
- * Advanced resource control (cpu, memory, network I/O, block I/O, disk usage and kernel resources)
- * Device passthrough (USB, GPU, unix character and block devices, NICs, disks and paths)
- * Network management (bridge creation and configuration, cross-host tunnels, ...)
- * Storage management (support for multiple storage backends, storage pools and storage volumes)
+## What is a system container?
+When running a system container, LXD simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
 
-# Integration with OpenNebula
-Starting from [OpenNebula EDGE](https://opennebula.org/get-your-hands-on-v-5-8-edge/) this cloud management platform packs official drivers to manage LXD compute nodes.
+### Application containers vs. system containers
+Application containers (as provided by, for example, Docker or Kubernetes) package a single process or application. System containers, on the other hand, simulate a full operating system and let you run multiple processes at the same time.
 
-Key features:
+Therefore, application containers are suitable to provide separate components, while system containers provide a full solution of libraries, applications, databases and so on. In addition, you can use system containers to create different user spaces and isolate all processes belonging to each user space, which is something you cannot do with application containers.
```

Comment:
Maybe we can do some rewording to avoid the "something you cannot do with application containers" just to avoid triggering those that will no doubt point that you can run an init system in a Docker container. It's just not what they are meant for and in general is a bad idea.

Reply:
Changed to "which is not what application containers are intended for."